app/src/main/java/com/example/bfinerocks/flashcard/constants/ConstantsForReference.java

```diff
@@ -0,0 +1,9 @@
+package com.example.bfinerocks.flashcard.constants;
+
+/**
+ * Created by BFineRocks on 12/30/14.
+ */
+public class ConstantsForReference {
+    public static final String USER_FIREBASE_REFERENCE = "firebase_user_ref";
+    public static final String USER_NAME_PREFERENCE = "preferredUser";
+}
```

Comment:
This is a good habit for keys and tags. The class itself could use a more descriptive name, though.